recipe/meta.yaml

```diff
@@ -44,24 +30,24 @@ requirements:
     - python
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - numpy 1.9.*
+    - numpy 1.13.*
```

Comment:
Generally we should take the opportunity to cleanup the pinnings here a bit. Namely `openblas` shouldn't need a hard coded pinning. Re-rendering after this should add in the pinnings. We can use the bot to do this.

ref: https://conda-forge.org/docs/meta.html#pinning-packages
ref: https://conda-forge.org/docs/meta.html#building-against-numpy
ref: https://conda-forge.org/docs/webservice.html#conda-forge-admin-please-rerender

Reply:
will look into this later. need to learn more about conda-forge a bit.